index.html

```diff
@@ -310,20 +309,26 @@ <h3>
           <ol>
             <li>
 Replace ":" with "/" in the method specific identifier to obtain the fully
-qualified domain name and optional path.
+qualified domain name, optional path, and optional filename.
```

Comment:
suggest removing the `filename` part from the current PR, and opening a new one once we have fixed the index issue.